imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -17,11 +18,84 @@ import (
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
 )
 
+// The values have been decided based on running spot tests for different images.
+var imageFamilyBootTimeThresholdMap = map[string]float64{
+	"centos":      60,
+	"debian":      50,
+	"rhel":        60,
+	"rocky-linux": 60,
+	"sles-12":     85,
+	"sles-15":     120,
+	"ubuntu":      75,
+	"ubuntu-pro":  110,
+}
+
 const (
 	markerFile     = "/boot-marker"
 	secureBootFile = "/sys/firmware/efi/efivars/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c"
 )
 
+func getThresholdValue(image string) float64 {
```

Comment:
i would rather we use explicit logic here as in other tests, rather than assuming we will always be able to create a name map that can return data using this ad hoc algorithm. this approach can easily introduce subtle bugs.

explicit logic means a switch/case statement checking images that we know we have support for.